helpers/containsUser.ts

```diff
@@ -0,0 +1,20 @@
+import { UserDto } from "../dto/userDto";
+import { UserDtoResponse } from "../dto/userDtoResponse ";
+
+export function containsUser(
+  obj: UserDto | UserDtoResponse,
+  list: UserDtoResponse[] | UserDto[],
+): boolean {
+  let i: number;
+  for (i = 0; i < list.length; i++) {
```

Comment:
you could write right here, if I am not mistaken: 
for(let i: number = 0; ..)

Yuo can not to write :number for variable which is an indexator for the cycle